### ISSUE TITLE
NewPipe can't be resized

### ISSUE THREAD

**Issue by alexanderadam** [TeamNewPipe/NewPipe#3947]
### Version
- [`0.19.5` from F-Droid](https://f-droid.org/en/packages/org.schabi.newpipe/#0.19.5)

### Steps to reproduce the bug

1. Open NewPipe app on an Android Desktop environment
2. Try to resize it

### Expected behavior

Resizing is possible

### Actual behaviour

It's not resizable via dragging borders or corners and the resizing button is greyed out

### Screenshots/Screen recordings

As you can see the resizing button is greyed out:
![NewPipe on Samsung DeX](https://user-images.githubusercontent.com/372620/88523995-16876600-cff9-11ea-9462-8084d886ba03.jpg)

### Additional context

I'm using [Samsung DeX](https://www.samsung.com/us/explore/dex/), which gives a desktop like experience and makes use of Android windowing support.

[Samsung has documentation on how to enable window resizing](https://developer.samsung.com/samsung-dex/modify-optimizing.html):

```xml
<application
    android:resizeableActivity="true">
</application>
```

This documentation also has more tweaks and information on how to improve the desktop experience in case you're interested.
They also have [an app testing guide](https://developer.samsung.com/samsung-dex/testing.html) (although some apps are fine with this guide but won't work on "real" DeX sometimes).

Applications like Firefox, VLC or Termux can be resized without any problems.

**PS:** I know that it is possible to _force_ all apps to be resizable via `Samsung Labs` > `Force Resize` but it would obviously better to make the application natively resizable to create less friction.
**PPS:** I'm not sure whether this is relevant but [this](https://codeberg.org/gitnex/GitNex/pulls/601/files) is how GitNex fixed the issue.
**PPPS:** There's also [this blog post](https://blog.iamsuleiman.com/prepare-apps-multi-window-in-android-n/) that looks helpful.

**Comment by TobiGr** [comment]
Can you test this APK, please: 
[NewPipe_basic_resize_APK.zip](https://github.com/TeamNewPipe/NewPipe/files/4981662/NewPipe_basic_resize_APK.zip)

**Comment by alexanderadam** [comment]
Hi @TobiGr,

1. You. are. insanely. fast. — seriously: you are awesome!
2. This apk works flawlessly fine for me. This release supports fluent resizing even with borders and corners.
3. Thank you on your general work on NewPipe, too! :+1:

**Comment by TobiGr** [comment]
1. You provided a feature request with details and a proper implementation. I just copied to code from the referenced commit and read the linked docs.
2. Nice. we still need to test compatibility with KitKat.
4. I gave you a debug version, so yes. The changes will be included in either 0.19.8 or 0.20.0,  but not in 0.19.7.
